### DISCUSSION TITLE
O que está faltando na documentação do Djàvue?

### DISCUSSION THREAD

**Discussion by huogerac** [evolutio/djavue3#39] [Q&A]
## 👋 Bem-vindo ao Djàvue, nos ajude a deixá-lo melhor!

Estamos utilizando o 'Discussions' deste repo aqui para entender melhor as dificuldades de quem começou ou tentou utilizá-lo
  * Que tipo ou qual documentação você mais sentiu falta, de arquitetura, de api ou dos setup?
  * Como rodar com docker, como rodar sem docker?
  * Qualquer informação sobre sua experiência será importante para a evolução deste template 💪

Para iniciar, basta deixar um comentário aqui em abaixo ⬇️.

**Comment by MarcosGrandesso** [comment]
um "how to" de como fazer deploy seria maneirissimo.

**Reply by Gabriels999** [reply]
Eu concordo com o que o @MarcosGrandesso apontou, só tenho receio de que ter isso por escrito poderia rapidamente ficar ultrapassado, porque depende da frequência com que a plataforma de deploy atualiza (nesse caso imagino que o fly.io)

**Reply by huogerac** [reply]
Mas como poderia ser então? Alguma sugestão?

**Reply by Gabriels999** [reply]
Ah, quando eu digo que concordo é porque seria bem legal, mas hoje não sei como seria feito para manter atualizado sem ser um trabalho bem chato

**Comment by luxu** [comment]
Roda "sem" docker para user Windows seria maravilhoso, tem como uma instalação nesse sentido Roger?

**Reply by huogerac** [reply]
Faz sentido sim! Preciso de ajuda de alguém tentar rodar no Windows para ver os erros!

**Comment by huogerac** [comment]
Quando usamos o Poetry por exemplo, a documentação fica desatualizada e não temos os passos necessários para baixar as dependencias com poetry...
